.github/CONTRIBUTING.md

```diff
@@ -116,7 +116,7 @@ We are lucky to have lots of [nice people][getting-help] here.
 #### Your First Code Contribution
 
 If you're looking for a good issue, you can look through
-the [available][labels-available] issues.
```

Comment:
any reason not to update the variable as well?

Reply:
I did chage it to read 'help wanted' instead of 'available'.  Is there another variable I can change?

<img width="639" alt="screen shot 2018-09-11 at 2 14 08 pm" src="https://user-images.githubusercontent.com/5035181/45388203-28aab780-b5cd-11e8-9cf6-cb554786da45.png">

Reply:
I tried doing `[help wanted][labels-help-wanted]` but it ends up like this:
<img width="491" alt="screen shot 2018-09-11 at 4 12 52 pm" src="https://user-images.githubusercontent.com/5035181/45392727-91019500-b5dd-11e8-8594-98148ebbbf5d.png">
I tried other ways of quoting and escaping the space in "help wanted" but couldn't get any to work.  Do you know how to link to a label that contains a space? 

Also the [url link](https://github.com/devtools-html/debugger.html/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22+-label%3A%22in+progress%22) I used filters out any issue tagged with "in progress" so you only see issues that are available.  I don't know how to do that with the link to a label either.

Reply:
You also need to declare the variable `labels-help-wanted` at almost bottom of the page, where other links are defined,  search for `labels-available`.

Reply:
@AnshulMalik thanks for that tip.  I was unaware md files can use variables.  Now I know :white_check_mark: